src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const changesArray = [];
```

Comment:
The `changesArray` is correctly initialized to store the state changes after each action is applied.